functional/vfio/run.sh

```diff
@@ -51,6 +54,11 @@ bind_to_vfio() {
 	echo "${dev_vendor_id}" | sudo tee "/sys/bus/pci/drivers/vfio-pci/new_id"
 }
 
+unbind_from_vfio() {
+	dev_vendor_id="$1"
+	echo "${dev_vendor_id}" | sudo tee "/sys/bus/pci/drivers/vfio-pci/remove_id"
+}
```

Comment:
Doing the rebinding by writing to `new_id` and `remove_id` isn't ideal.  Because it's based on the product IDs, there are messy edge cases if you have multiple devices of the same type in the system.

A better approach is to write (then clear) the driver name in the `driver_override` file in the device's sysfs directory.  That will  force attach that specific device instance to the driver, regardless of what product ids the driver advertises.  That should make the rebind code both smaller and more robust.

That's the approach used by the `driverctl` tool I mentioned on our last call.  Using that would be even better... but only if it's already packaged in all the distros we're using in the CI.

Reply:
I'll try to see how easy is to install the driverctl tool, thanks